protos/flyteidl/admin/matchable_resource.proto

```diff
@@ -64,3 +64,25 @@ message MatchingAttributes {
     ExecutionClusterLabel execution_cluster_label = 4;
   }
 }
+
+// Represents a custom set of attributes applied for either a domain; a domain and project; or
+// domain, project and workflow name.
+message MatchableResourceConfiguration {
+    MatchingAttributes attributes = 1;
+
+    string domain = 2;
+
+    string project = 3;
+
+    string workflow = 4;
```

Comment:
You would need Priority and Launch plan as well

Reply:
+1 for launch plan, thanks for the catch
as for priority, i'd rather the server render the resolved state of the world so the caller of this endpoint can see what the actual applied values are